etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-hosts.json

```diff
@@ -213,15 +215,16 @@
                 ],
                 "datasource": "",
                 "decimals": 1,
-                "format": "none",
+                "description": "Sum of Memory Free and Memory Cached.",
+                "format": "decbytes",
                 "gauge": {
                     "maxValue": 100,
                     "minValue": 0,
-                    "show": true,
+                    "show": false,
                     "thresholdLabels": false,
                     "thresholdMarkers": true
                 },
-                "id": 16,
+                "id": 34,
```

Comment:
Why is this required?

Reply:
@cloudbehl

Reply:
@rishubhjain Panel Id, To uniquely identify panels.